gtubt/lib/ui/pages/login_form.dart

```diff
@@ -131,7 +132,12 @@ class _LoginFormState extends State<LoginForm> {
       height: 40,
       margin: EdgeInsets.only(bottom: 32),
       child: RaisedButton(
-        onPressed: () => _isLoginButtonEnabled() ? _onFormSubmitted() : null,
+        onPressed: () {
+          if(_isLoginButtonEnabled()){
+            NotificationFactory.loadingFactory(message: "Logging in...").show(context);
```

Comment:
state degisimini kontrol edip BlocConsumer'in listener'inda yapilmali. `state.isLoading` kontrol edebilirsin.